GitUI/Editor/FileViewer.cs

```diff
@@ -216,7 +216,7 @@ public Encoding Encoding
                     }
                     else
                     {
-                        encodingToolStripComboBox.SelectedIndex = -1;
+                        encodingToolStripComboBox.SelectedIndex = 0;
```

Comment:
Should be submitted in a separate PR, for #5575

Reply:
I get exception at startup due to this:
System.ArgumentOutOfRangeException: InvalidArgument=Value på 0 är inte giltigt för SelectedIndex.
Parameternamn: SelectedIndex
   vid System.Windows.Forms.ComboBox.set_SelectedIndex(Int32 value)
   vid GitUI.Editor.FileViewer.<set_Encoding>b__64_0() i F:\dev\gc\gitextensions_4\GitUI\Editor\FileViewer.cs:rad 219
   vid GitUI.GitUIExtensions.<InvokeAsync>d__11.MoveNext() i F:\dev\gc\gitextensions_4\GitUI\GitUIExtensions.cs:rad 208
--- Slut på stackspårningen från föregående plats där ett undantag utlöstes ---
   vid System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   vid System.Runtime.CompilerServices.ConfiguredTaskAwaitable.ConfiguredTaskAwaiter.GetResult()
   vid GitUI.ThreadHelper.<>c__DisplayClass13_0.<<FileAndForget>b__0>d.MoveNext() i F:\dev\gc\gitextensions_4\GitExtUtils\GitUI\ThreadHelper.cs:rad 101

Reply:
Will fix this and submit another PR.